tests/tests/test_train_schedule.py

```diff
@@ -64,23 +64,13 @@ def test_editoast_get_and_update_schedule_result(west_to_south_east_simulation:
     assert "eco" in simulation_report
 
 
-def test_api_bulk_delete(small_scenario, west_to_south_east_simulations: Sequence[int]):
```

Comment:
The previous test was duplicated, right?

Reply:
Yes !